node-gui/src/main_window/main_widget/tabs/wallet/stake.rs

```diff
@@ -61,59 +61,72 @@ const CREATE_STAKING_POOL_TOOLTIP_TEXT: &str = "A staking pool locks the pledge
 const START_STAKING_TOOLTIP_TEXT: &str =
     "If you have created pools, this will activate staking. The node must be kept running, in order to assist the network in creating blocks and earn rewards.";
 
+const DECOMMISSION_POOL_ID_TOOLTIP_TEXT: &str = "The pool ID to decommission.";
```

Comment:
This is the pool address. We don't use the term "id" for the public, especially that this has to be bech32.

DECOMMISSION_POOL_ADDRESS_TOOLTIP_TEXT

Also, the text itself: "The address of the pool that will be decommissioned. Make sure to stop staking before going through with this."